tests/Scoper/PhpScoperTest.php

```diff
@@ -965,5 +965,114 @@ function foo($bar) : Closure
 
 PHP
         ];
+
+        //
+        // Function parameters
+        //
+        // ====================
+
+        yield '[Function parameter] class_exists' => [
+            <<<'PHP'
+<?php
+
+class_exists('Symfony\\Component\\Yaml\\Yaml');
+
+PHP
+            ,
+            'Humbug',
+            <<<'PHP'
+<?php
+
+class_exists('Humbug\\Symfony\\Component\\Yaml\\Yaml');
+
+PHP
+        ];
+
+        yield '[Function parameter] interface_exists' => [
+            <<<'PHP'
+<?php
+
+interface_exists('Foo\\Bar');
+
+PHP
+            ,
+            'Humbug',
+            <<<'PHP'
+<?php
+
+interface_exists('Humbug\\Foo\\Bar');
+
+PHP
+        ];
+
+        yield '[Function parameter] class_exists with concat strings' => [
+            <<<'PHP'
+<?php
+
+class_exists('Symfony\\Component' . '\\Yaml\\Yaml');
+
+PHP
+            ,
+            'Humbug',
+            <<<'PHP'
+<?php
+
+class_exists('Symfony\\Component' . '\\Yaml\\Yaml');
+
+PHP
+        ];
+
+        yield '[Function parameter] class_exists with constant' => [
+            <<<'PHP'
+<?php
+
+class_exists(\Symfony\Component\Yaml\Yaml::class);
+
+PHP
+            ,
+            'Humbug',
+            <<<'PHP'
+<?php
+
+class_exists(\Humbug\Symfony\Component\Yaml\Yaml::class);
+
+PHP
+        ];
+
+        yield '[Function parameter] class_exists with variable (no change)' => [
+            <<<'PHP'
+<?php
+
+$x = '\\Symfony\\Component\\Yaml\\Yaml';
+class_exists($x);
+
+PHP
+            ,
+            'Humbug',
+            <<<'PHP'
+<?php
+
+$x = '\\Symfony\\Component\\Yaml\\Yaml';
+class_exists($x);
+
+PHP
+        ];
+
+        yield '[Function parameter] Skip when prefix is the same' => [
+            <<<'PHP'
+<?php
+
+class_exists('Humbug');
+
+PHP
+            ,
+            'Humbug',
+            <<<'PHP'
+<?php
+
+class_exists('Humbug');
```

Comment:
this one may look a bit weird because the class in question is the same as the prefix. Just for the sake of clarity I would pick another name `Foo` or `PhpScoper`